db/schema.rb

```diff
@@ -37,8 +37,6 @@
     t.integer "people_goal"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["end_date", "start_date"], name: "between_end_start_dates"
```

Comment:
These should exist. I'm not sure why your branch doesn't have them. Maybe un-track this file?

Reply:
Same for all in file.